src/fdc/DiskManipulator.cc

```diff
@@ -406,7 +409,7 @@ void DiskManipulator::create(std::span<const TclObject> tokens)
 	if (sizes.empty()) {
 		throw CommandException("No size(s) given.");
 	}
-	if (sizes.size() > 1) {
+	if (sizes.size() > 1 && bootType != MSXBootSectorType::NEXTOR) {
 		// extra sector for partition table
 		++totalSectors;
 	}
```

Comment:
This is unexpected to me.
I understand that for a Sunrise partition table there's only one extra sector for the whole disk. While for Nextor there's one extra sector per partition (right?).
But then I would have expected to see: `totalSectors += sizes.size()`.

Reply:
Note partitionNextor does:

    p.start = 1;
    p.size = size - 1;

Reply:
Makes sense.

Though I should have been more clear: my remark was not so much about the missing `totalSectors += ...` stuff. But about the difference between how Nextor and Sunrise partition tables are accounted. So instead of adding `totalSectors += ...` for Nextor we could also make it uniform by removing `++totalSectors` for Sunrise.

I understand that the latter is your plan?

Reply:
Yeah, that’s my intention.